### PR TITLE
Minor clean-up: don't use Assert.notNull

### PR DESCRIPTION
**Pull request by bgrozev**
Uses Objects.requireNonNull instead of Assert.notNull.

### CODE REVIEW

File: src/main/java/org/jitsi/impl/protocol/xmpp/OpSetSubscriptionImpl.java [hunk 1]
```diff
@@ -311,10 +311,9 @@ public void handlePublishedItems(ItemPublishEvent event)
          */
         public Subscription(String node, SubscriptionListener listener)
         {
-            Assert.notNull(node, "node");
-            Assert.notNull(listener, "listener");
+            Objects.requireNonNull(listener, "listener");
+            this.node = Objects.requireNonNull(node, "node");
 
-            this.node = node;
             this.listeners.add(listener);
```
Comment (paweldomas) [inline src/main/java/org/jitsi/impl/protocol/xmpp/OpSetSubscriptionImpl.java]: Will this work here this.listeners.add(Objects.requireNotNull(listener, "listener") ?

File: src/main/java/org/jitsi/jicofo/rest/Health.java [hunk 2]
```diff
@@ -253,10 +253,7 @@ private static int cacheStatus(int status)
         JitsiMeetServices services
             = focusManager.getJitsiMeetServices();
 
-        Assert.notNull(services, "services");
```
Comment (paweldomas) [inline src/main/java/org/jitsi/jicofo/rest/Health.java]: The reason I've put those here was to get a message in NPE indicating which variable is null(to know even without looking at the line number and checking the code). I don't insist that those are necessary just explaining my reasoning behind that. If you think it's not of much use I'm good with having those deleted.

Comment (bgrozev) [inline src/main/java/org/jitsi/jicofo/rest/Health.java]: OK, they looked superfluous to me, but that makes sense. Will add them back.

File: src/main/java/org/jitsi/impl/protocol/xmpp/OpSetSubscriptionImpl.java (hunk 1 above)
Comment (bgrozev) [inline src/main/java/org/jitsi/impl/protocol/xmpp/OpSetSubscriptionImpl.java]: Yes, but I find it harder to read.

**Pull request merged**